anautilities.py

```diff
@@ -41,6 +42,17 @@ def initVFATArray(array_dtype, nstrips=128):
 
     return np.zeros(nstrips, dtype=list_dtypeTuple)
 
+def make3x8Canvas(name, initialContent = None, drawOption = ''):
```

Comment:
Can you add a comment above this which describes what `initialContent` is expected to be?  Looks like some dict or list of TObjects that have a `Draw()` method but this would clarify things.